modules/node-observability-scripting-cr.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-observability-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="node-observability-scripting-cr_{context}"]
+= Creating the Node Observability custom resource for scripting
+
+You must create and run the `NodeObservability` custom resource (CR) before you run the scripting. When you run the `NodeObservability` CR, it creates the necessary machine config and machine config pool CRs to enable the scripting on the worker nodes matching the `nodeSelector`.
```

Comment:
Could you change this too something like (as it does not make use of machine config and machine config pools)

```
You must create and run the `NodeObservability` custom resource (CR) before you run the scripting. When you run the `NodeObservability` CR, it will enable the agent in scripting mode on the worker nodes matching the `nodeSelector`. 
```
```

Reply:
Right, so, I should rephrase it to explain there will be no machine config and machine config pods? Correct me if I am wrong, custom resource is to be created for scripting as well right?

Reply:
> Right, so, I should rephrase it to explain there will be no machine config and machine config pods? Correct me if I am wrong, custom resource is to be created for scripting as well right?

Yes correct when you use the field "type" : "scripting" it does not make use of machine config and machine config pools. Yes we create a CR for both, the only difference is we use "type": "crio-kubelet" for CRIO profiling and "type": "scripting" to enable and execute the embedded scripts. Notice we can't do both at the same time, we will have to re-deploy to change from crio-kubelet to scripting and vice-versa

Reply:
Understood.